src/v0/sources/revenuecat/transform.js

```diff
@@ -0,0 +1,49 @@
+const { camelCase } = require('lodash');
+const path = require('path');
+const fs = require('fs');
+const moment = require('moment');
+const { removeUndefinedAndNullValues, isDefinedAndNotNull } = require('../../util');
+const Message = require('../message');
+
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(fs.readFileSync(path.resolve(__dirname, './mapping.json'), 'utf-8'));
```

Comment:
I guess we can import `json` directly. Can you once check and confirm ?

Reply:
Removed mapping itself as it deemed unnecessary to have it for two very simple mappings.